Jenkinsfile

```diff
@@ -0,0 +1,337 @@
+                            junit '**/nosetests.xml'
+                        }
+                    }
+                }
+                stage('commonlib_unit_2') {
+                    agent { label "jenkins-worker" }
+                    environment {
+                        SHARD = 2
+                        TEST_SUITE = 'commonlib-unit'
+                    }
+                    steps {
+                        script {
+                            runPythonTests()
+                        }
+                    }
+                    post {
+                        always {
+                            archiveArtifacts allowEmptyArchive: true, artifacts: 'reports/**/*,test_root/log/**/*.log,**/nosetests.xml,*.log'
+                            junit '**/nosetests.xml'
+                        }
+                    }
+                }
+                stage('commonlib_unit_3') {
+                    agent { label "jenkins-worker" }
+                    environment {
+                        SHARD = 3
+                        TEST_SUITE = 'commonlib-unit'
+                    }
+                    steps {
+                        script {
+                            runPythonTests()
+                        }
+                    }
+                    post {
+                        always {
+                            archiveArtifacts allowEmptyArchive: true, artifacts: 'reports/**/*,test_root/log/**/*.log,**/nosetests.xml,*.log'
+                            junit '**/nosetests.xml'
+                        }
+                    }
+                }
+            }
+        }
+        stage('Run coverage') {
+            environment {
+                CODE_COV_TOKEN = credentials('CODE_COV_TOKEN')
+                TARGET_BRANCH = "origin/master"
+                CI_BRANCH = "${ghprbSourceBranch}"
+                SUBSET_JOB = "null" // Keep this variable until we can remove the $SUBSET_JOB path from .coveragerc
+            }
+            steps {
+                ansiColor('gnome-terminal') {
+                    sshagent(credentials: ['jenkins-worker'], ignoreMissing: true) {
+                        checkout changelog: false, poll: false, scm: [$class: 'GitSCM', branches: [[name: '${sha1}']],
+                            doGenerateSubmoduleConfigurations: false, extensions: [], submoduleCfg: [],
+                            userRemoteConfigs: [[credentialsId: 'jenkins-worker',
+                            refspec: '+refs/heads/*:refs/remotes/origin/* +refs/pull/*:refs/remotes/origin/pr/*',
+                            url: 'git@github.com:edx/edx-platform.git']]]
+                        unstash 'lms-unit-1-reports'
+                        unstash 'lms-unit-2-reports'
+                        unstash 'lms-unit-3-reports'
+                        unstash 'lms-unit-4-reports'
+                        unstash 'cms-unit-1-reports'
+                        unstash 'commonlib-unit-1-reports'
```

Comment:
It looks like you still need to account for the new shards in the coverage data collection.

Reply:
ah, good catch